### ISSUE TITLE
Flags mutually exclusive but one of them required

### ISSUE THREAD

**Issue by krystian-panek-wttech** [spf13/cobra#1891]
hey, thanks for cool library, I have only a one question:

```go
func pkgDefineFileAndUrlFlags(cmd *cobra.Command) {
	cmd.Flags().String("file", "", "Local ZIP path")
	cmd.Flags().String("url", "", "URL to ZIP")
	cmd.MarkFlagsMutuallyExclusive("file", "url")
}
```

how to make only one of flags defined required, I mean `--file` or `--url` ? but not at the same time?

I see that there is `MarkFlagsRequiredTogether` but I more want to have sth like `MarkFlagsRequiredSome("file", "url")` :)

Any ideas have to handle such case?
Greetings, Krystian

**Comment by marckhouzam** [comment]
Sadly this is not supported. See #1761 .
You would have to implement that logic yourself in your program once the flags are parsed.

**Comment by krystian-panek-wttech** [comment]
ok, thanks for confirmation; I hope I will find some time to improve it in Cobra someday :)

**Comment by marckhouzam** [comment]
> ok, thanks for confirmation; I hope I will find some time to improve it in Cobra someday :)

That would be great!

**Comment by marckhouzam** [comment]
Fixed by @marevers in #1952
